app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetControllerV2.kt

```diff
@@ -363,19 +367,10 @@ class BloodPressureEntrySheetControllerV2 @Inject constructor(
         .ofType<Valid>()
         .map { it.parsedDate }
 
-    data class BP(val systolic: Int, val diastolic: Int)
-
-    val systolicChanges = events
-        .ofType<BloodPressureSystolicTextChanged>()
-        .map { it.systolic }
-
-    val diastolicChanges = events
-        .ofType<BloodPressureDiastolicTextChanged>()
-        .map { it.diastolic }
-
-    val bpChanges = validDates
-        .withLatestFrom(systolicChanges, diastolicChanges)
-        .map { (_, systolic, diastolic) -> BP(systolic.toInt(), diastolic.toInt()) }
+    val validBps = events
+        .ofType<BloodPressureBpValidated>()
```

Comment:
The name `BloodPressureBpValidated` is a little redundant, don't you think? If the events had been prefixed with `BloodPressureEntry`, it might have made sense but it feels a little weird now.

Reply:
I know 😅 I did this to keep it consistent with `BloodPressureDateValidated`. Otherwise we could have `BloodPressureValidated` and `BloodPressureDateValidated`.

What do you think about `BloodPressureReadingsValidated`?

Reply:
Works.